test/integration/relay/connection.test.js

```diff
@@ -117,26 +117,22 @@ describe('relay', function () {
           values: {
             ID: {value: [this.Task.primaryKeyAttribute, 'ASC']},
             LATEST: {value: ['createdAt', 'DESC']},
-            CUSTOM: {value: ['updatedAt', 'DESC']},
-            NAME: {value: ['name', 'ASC']}
-          }
-        }),
-        before: (options) => {
-          if (options.order && options.order[0][0] === 'updatedAt') {
-            if (sequelize.dialect.name === 'postgres') {
-              options.order = Sequelize.literal(`
+            CUSTOM: {value: [
+              Sequelize.literal(sequelize.dialect.name === 'postgres'
+                ? `
                 CASE
                   WHEN completed = true THEN "createdAt"
-                  ELSE "otherDate" End ASC`);
-            } else {
-              options.order = Sequelize.literal(`
+                  ELSE "otherDate" End`
+                : `
                 CASE
                   WHEN completed = true THEN \`createdAt\`
-                  ELSE \`otherDate\` End ASC`);
-            }
+                  ELSE \`otherDate\` End`
```

Comment:
`sequelize.dialect.QueryGenerator.quoteIdentifier()`

Reply:
@janmeier that's good to know, thanks for the tip!